build.xml

```diff
@@ -146,6 +146,18 @@
 
     </target>
 
+    <target name="check-ca" if="cacert.path">
```

Comment:
This seems like it would be easier to just use `<copy>` combined with `failonerror=false, quiet=true` in our standard include-assets task.

In regards to properly appending to the property file, we should be able to shim that logic into the respective keygen logic via something like `if ${cacert.path} != '' // write it out` logic (adapted for each keygen).

Although I understand that the `ca.certificate` has nothing at all to do with the `wss` key generation task, it's where we create the `qz-tray.properties` file, so it prevents the `>` vs `>>` problems.